prisma/schema.prisma

```diff
@@ -70,39 +71,53 @@ enum UserPlan {
   PRO
 }
 
+model DestinationCalendar {
+  id          Int        @id @default(autoincrement())
+  integration String
+  externalId  String
+  user        User?      @relation(fields: [userId], references: [id])
+  userId      Int?       @unique
+  booking     Booking?   @relation(fields: [bookingId], references: [id])
+  bookingId   Int?       @unique
+  eventType   EventType? @relation(fields: [eventTypeId], references: [id])
+  eventTypeId Int?       @unique
```

Comment:
We may override the user destination calendar and choose per event type in the future.